Assets/FOVMapping/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "com.stupasoft.fovmapping",
+  "displayName": "Fog of War and Field of View Mapping",
+  "version": "2.1.0",
+  "unity": "2022.3",
+  "licensesUrl": "See LICENSE.md",
+  "hideInEditor": "false",
+  "author": {
+    "name": "StupaSoft",
+    "email": "stupasoft@gmail.com"
+  },
+  "description": "Fast GPU accelerated FOV mapping for RTS Fog of War Generation for Unity.",
+  "dependencies": {},
+  "keywords": [
```

`NativeQueue<>` requires the **Collections** package as a dependency.
```suggestion
  "dependencies": {
    "com.unity.collections": "2.5.3"
  },
  "keywords": [
```

---

Good catch, 
the editor asmdef references 3 unity libraries, I have added all three with the latest versions I have running in my project, but feel free to suggest other versions
``` 
 "dependencies": {
        "com.unity.collections": "2.5.3",
        "com.unity.mathematics": "1.3.2",
        "com.unity.burst": "1.8.21"
  },
```